src/http_server.cc

```diff
@@ -1996,7 +2000,29 @@ HTTPAPIServer::HandleTrace(evhtp_request_t* req, const std::string& model_name)
       trace_response.AddString("log_frequency", std::to_string(log_frequency)));
   RETURN_AND_RESPOND_IF_ERR(
       req, trace_response.AddString("trace_file", filepath));
-
+  RETURN_AND_RESPOND_IF_ERR(
+      req,
+      trace_response.AddString(
+          "trace_mode", trace_manager_->InferenceTraceModeString(trace_mode)));
+  auto mode_key = std::to_string(trace_mode);
+  auto trace_options_it = config_map.find(mode_key);
```

Comment:
Food for thought in the future with Olga, but I found this code misleading to read.

Upon reading it, I naively thought `std::to_tsring(trace_mode)` would be something like `"TRITON"` or `"OPENTELEMETRY"`, but it's actually "0" or "1" based on order in the enum. `InferenceTraceModeString` is the one that actually converts to a string. TLDR; I found it confusing that the keys in this map are "0" and "1" rather than the actual string versions of the modes.

No action here, just mentioning something I found misleading to think about down the line.